docs/documentation/advanced_examples/monitored-oscillator.md

```diff
@@ -0,0 +1,115 @@
+# Continuous measurement of an oscillator
```

Comment:
I think these two tutorials are great as is, but I would suggest merging them into a single "Continuous measurement" tutorial with two main sections (Oscillator and Qubit) since they explain exactly the same thing simply on two different systems.